aws-xray-recorder-sdk-sqs/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+plugins {
+    `java-library`
+    `maven-publish`
+}
+
+buildscript {
+    dependencies {
+       classpath("com.amazonaws:aws-lambda-java-events:3.11.0")
+    }
+}
+
+dependencies {
+    compileOnly("com.amazonaws:aws-lambda-java-events:3.11.0")
+}
+
+tasks.jar {
+    manifest {
+        attributes("Automatic-Module-Name" to "com.amazonaws.xray_sqs")
```

Comment:
Module name could be sqs helper since it is not for sqs instrumentation

Reply:
Done